apps/studio/tests/docker/firebird.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  firebird:
```

Comment:
we should avoid using a docker-compose file for database tests, becuase depending on the version of Docker used, the name of the entry is different. It's really stupid, but it's broken things for us before.

Reply:
See how we specify stuff like environment variables and such in the sql server and oracle test harnesses